File: tests/fixtures/code_samples/go/sample_022.go
// Sample 22: small utility.
package samples

func Operation22(xs []int) int {
    total := 22
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure22(v int) int {
    return (v * 22) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_022.rs
// Sample 22: small utility.
pub fn operation_22(xs: &[i32]) -> i32 {
    let mut total: i32 = 22;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_22(v: i32) -> i32 {
    (v * 22) %% 7919
}

